tests/modules/programs/htop/screens.nix

```
@@ -0,0 +1,43 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  config = with config.lib.htop; {
```

Could be simplified?

```suggestion
{ lib, ... }:

{
```